iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -81,15 +103,20 @@ class ConversationMessagesCell: UICollectionViewCell {
     }
 
     /// Configures the cell to display the given messages. The message sequence should be ordered newest to oldest.
-    func set(sequence: MessageSequence) {
-        self.conversation = sequence
+    func set(conversation: Conversation) {
+        // Create a new conversation controller if this is a different conversation than before.
+        if conversation.cid != self.conversation?.cid {
+            self.conversationController = ChatClient.shared.channelController(for: conversation.cid)
+            self.subscribeToUpdates()
+        }
 
         // Scroll to the last item when a new conversation is loaded.
         if self.dataSource.snapshot().itemIdentifiers.isEmpty {
             self.scrollToLastItemOnLayout = true
             self.setNeedsLayout()
         }
-        self.dataSource.set(messageSequence: sequence)
+
+        self.dataSource.set(messageSequence: conversation, showLoadMore: self.shouldShowLoadMore)
```

Comment:
The message sequence datasource now requires that you specify if the load more cell is shown.

Unfortunately the conversation object itself doesn't have enough information to say one way or the other.